### DISCUSSION TITLE
[Fine-grained personal access tokens] Split content permission to multi permissions

### DISCUSSION THREAD

**Discussion by M97Chahboun** [community/community#47686] [Apps, API and Webhooks]
### Select Topic Area
Question

### Body

on Fine-grained personal access tokens Why not split content permission to multi permissions
as example :
- Releases permission (read/write)
- Commits permission (read/write)
...

**Comment by ento** [comment]
This will also be useful for [GitHub Actions job permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) (which may come for 'free' if the underlying permission system is the same). I'd rather not give a job permission to mess with the Git content of my repo when I only want to let it create releases. One manageable way to split this could be to split in two, Git content (commits, branches, etc) and GitHub content (releases, code-scanning, dependency-graph, etc).
